src/utils.js

```diff
@@ -48,3 +48,8 @@ export const FORM = {
 	GROUP_FOLDER: 3,
 	APP_PASSWORD: 4,
 }
+
+export const SEARCHFROM = {
```

Comment:
How about something like this?
```suggestion
export const SEARCH_WORKPACKAGES_FROM = {
```